dotcom-rendering/src/web/components/SupportingContent.tsx

```diff
@@ -64,7 +65,8 @@ export const SupportingContent = ({ supportingContent, alignment }: Props) => {
 			{supportingContent.map((subLink: DCRSupportingContent, index) => {
 				// The model has this property as optional but it is very likely
 				// to exist
-				if (!subLink.headline) return <></>;
```

Comment:
Would you still need an ID if we returned null here? I can't remember if this has caused us issues before